applications/OptimizationApplication/python_scripts/controls/shape/vertex_morphing_shape_control.py

```diff
@@ -150,29 +36,16 @@ def __init__(self, name: str, model: Kratos.Model, parameters: Kratos.Parameters
 
         default_settings = Kratos.Parameters("""{
             "controlled_model_part_names": [],
-            "filter_settings": {
-                "type" : "surface_explicit",
-                "filter_function_type" : "linear",
-                "damping_function_type" : "sigmoidal",
-                "radius": 0.000000000001,
-                "max_nodes_in_filter_radius": 1000,
-                "linear_solver_settings" : {}
-            },
-            "mesh_motion_settings" : {},
-            "output_all_fields": false,
-            "fixed_model_parts": {},
-            "utilities": []
+            "filter_settings"            : {},
+            "use_mesh_motion_solver"     : false,
+            "skin_model_part_names"      : [],
```

Comment:
skin_model_part_names is not self explaining please change it to sth like fixed_model_parts and also get the components for fixing

Reply:
will be done.